src/.nuget/packaging.props

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <!-- Packaging projects are non-SDK-style and directly import Directory.Build.props. They can also import Microsoft.Common.props.
```

Comment:
It seems like there had to be a reason to go through this pain. Why are these not just converted to sdk projects?

Reply:
For the pkgproj ones, there isn't any natural SDK-style support for they way they are built (at least not that I could find). If they were to be converted, it would require awkwardly working around the system to make it do what we want (it would pull in some targets which get chained into build which we don't want / would have to get around, we'd need to specify custom after targets to pull in the packaging targets after the Directory.Build.targets). Explicitly pulling in Directory.Build files seemed simpler in this case, since that is actually what we want

Since I wasn't switching the pkgproj, I left all of them non-SDK-style. I think I should be able to change `packages.builds` easily and probably the other .builds files (i.e. the traversal ones, which should be enough avoid this potential double import)

Reply:
Even for CoreFX, this is one of the few projects that remains as a non-SDK project. I'd have to take a look, but that alone seems enough to hint at it not being the simplest task. I'll ask Eric St. John when I get a chance, but I feel that might be a task for another PR.

Reply:
Sounds good, marking resolved